templates/katello.yaml.erb

```diff
@@ -33,7 +33,9 @@
     :url: <%= @qpid_url %>
     :subscriptions_queue_address: <%= @candlepin_event_queue %>
 
+  <%- if @use_pulp_2_for_docker -%>
```

Comment:
Should this be a line higher so you don't have an empty line at the end of the file?